src/backend/distributed/shared_library_init.c

```diff
@@ -654,7 +654,7 @@ RegisterCitusConfigVariables(void)
 					 "the background worker moves on. When set to -1 this background "
 					 "process is skipped."),
 		&DeferShardDeleteInterval,
-		-1, -1, 7 * 24 * 3600 * 1000,
+		60000, -1, 7 * 24 * 3600 * 1000,
```

Comment:
I think we can make the default 15 seconds because it might be confusing for users to see that there are still old shards even though they did a rebalance.

Reply:
I have found a minute to be tediously long when testing things.

Reply:
Yeah a minute is long, is 15 seconds also long?

Reply:
I'd be ok with 15 seconds. I think shorter might be problematic when things are failing.